commonv5/ts/helpers/constants.ts

```diff
@@ -38,6 +38,9 @@ export enum AzureBuildVariables {
 export enum TaskVariables {
   SonarQubeServerVersion = "SONARQUBE_SERVER_VERSION",
   SonarQubeScannerMode = "SONARQUBE_SCANNER_MODE",
+  SonarQubeMsBuildVersion = "SONARQUBE_MSBUILD_VERSION",
+  SonarQubeCliVersion = "SONARQUBE_CLI_VERSION",
+  SonarQubeScannerLocation = "SONARQUBE_SCANNER_LOCATION",
```

Comment:
Now may be a good time to start renaming these variables to `SONAR_xx` rather than `SONARQUBE_xx`, since they are not SQ-dependant?